lib/cdo/delete_accounts_helper.rb

```diff
@@ -45,6 +45,13 @@ def delete_project_backed_progress(user)
       where(id: storage_app_ids).
       update(value: nil, updated_ip: '', updated_at: Time.now)
 
+    # Clear any comments associated with specific versions of projects.
+    # At time of writing, this feature is in use only in Javalab when a student
+    # commits their code.
+    project_versions = ProjectVersion.where(storage_app_id: storage_app_ids)
+    project_versions.each {|version| version.update!(comment: nil)}
+    @log.puts "Cleared #{project_versions.count} ProjectVersion" if project_versions.count > 0
```

Comment:
nit: should this instead say `Cleared x ProjectVersion comments`?

Reply:
In other places we clear comments we just use the model name, but in this case the model name is less descriptive. Just updated!